### DISCUSSION TITLE
diary/2023/10/25/principles-of-decision-making/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#149] [Comments]
# diary/2023/10/25/principles-of-decision-making/

키워드 : 의사 결정 프로세스, 의사 결정 방법, 의사 결정 과정, 의사 결정 원칙 우리는 살면서 의사 결정을 해야하는 경우가 많습니다. 이럴 때 어떻게 해야 최선의 의사 결정을 할 수 있을까?라는 생각을 하곤 합니다. 아마 많은 분들이 한번쯤 이런 생각을 하셨을거에요. 나심 니콜라스 탈레브의 안티프래질 책을 2021년에 읽고, 김창준님의 함께...

https://zzsza.github.io/diary/2023/10/25/principles-of-decision-making/

**Comment by Invidam** [comment]
> 다른 사람들이 어떤 선택을 하는지 보고, 일반적이지 않은 선택을 한다

신박하면서도 설득력있네요  ㅋㅋㅋ

**Reply by zzsza** [reply]
의도적으로 다른 각도로 틀다보면 그것이 포텐이 크게 생기는 경험을 해서 이런 원칙을 만들었어요..!

**Comment by cestbonciel** [comment]
분석하고 객관화하는 것이 중요하다고 느껴지는 글이네요. 북마크하고 또 읽어봐야겠어요 :) 잘읽었습니다. 감사합니다.

**Reply by zzsza** [reply]
읽어주셔서 감사합니다! 하나씩 확인하고 개선하는 과정이 있으면 도움이 되더라구요!

**Comment by NassimTalebGigerenzer** [comment]
안녕하세요.  글 잘 읽었습니다. 혹시 잠재력을 최대한 끌어내는 것에 대한 의견이 있으신가요?
